flat/lib/typescript.js

```diff
@@ -70,6 +69,21 @@ module.exports = function typescript() {
         "@typescript-eslint/no-confusing-non-null-assertion": "off",
         "@typescript-eslint/prefer-for-of": "off",
         "@typescript-eslint/prefer-function-type": "off",
+
+        // rules deleted from `recommended` in v8
+        "no-loss-of-precision": "off",
+        "@typescript-eslint/no-loss-of-precision": "error",
```

Comment:
It seems that the rule "@typescript-eslint/no-loss-of-precision" has been removed. Is this setting still necessary?

https://typescript-eslint.io/blog/announcing-typescript-eslint-v8/#rule-breaking-changes
https://github.com/typescript-eslint/typescript-eslint/pull/8832

Reply:
Thank you very much!
This rule seems to have been deprecated because the eslint rules have been enhanced and the ts-eslint rules are no longer needed, so it looks good to remove it as you pointed out!

c.f. : https://typescript-eslint.io/rules/no-loss-of-precision/